### ISSUE TITLE
Samples running trouble

### ISSUE THREAD

**Issue by biggorila** [NVIDIA/gvdb-voxels#5]
I have install the gvdb library but I can not make the samples to run. I can compile the samples .cpp and when the programs are running a CUDA error appear "CUDA ERROR: file not found (func: cuda_gvdb_module.ptx, caller: ModuleLoad)".

My guess is that the gvdb library trying to find the file but is not in the supposed direction. Since the installation is made on the system the address of the file is not properly defined. I can copy the file to the corresponding address, but I don't know where to put it the system, in order for the gvdb should look for.

**Comment by djspecialagent** [comment]
Do you have all the required ptx files in the directory where you are trying to run the sample? I had to copy all these in, in order to get passed the errors you are encountering.

**Comment by biggorila** [comment]
The problem was kind of solve, the solution is to install the examples in order to run them, never the less other issues appear and the examples can't be run.

**Comment by rchoetzlein** [comment]
On the linux the install instructions state "remember to do 'make install' to get all files". On windows, the files should be copied during post-build step of visual studio. Fyi, the install location of the gvdb will be indicated by the GVDB_INSTALL_PREFIX variable in the cmake setup.
If you have other build issues with samples, please describe in further detail and we'll take a look.

Regards,
Rama Hoetzlein
NVIDIA

**Comment by biggorila** [comment]
I did find the problem, or the problem that I describe, the "fix" was actually to install the examples. Nevertheless I couldn't  make them run any way. 

The main problem (at least for me) was that the GVDB needs the graph library to work and this library is not supported for SM=20 and sm=21 (I know that they are going to be deprecated) devices therefore neither the GVDB library either, is my believed that this should be in the library documentation when I try to installed I did look if it was possible to install on a 21 device, that was the starting for a long search.

 Finally when Iook for the dependencies that where missing I found that all came for the graph library did not compile and in found that the issue was my GPU device thats why I did not answered before my apologies
